src/tracker/encoder/os_net/os_net_encoder.py

```diff
@@ -1,30 +1,29 @@
 import os
-from typing import Dict, List
+import os.path as osp
+import pickle
 
-import numpy as np
+# not useful
+import sys
```

Comment:
Not sure what to think of the comment: if this is not useful (and `sys` doesn't seem to be used in this file), why import it at all? I wonder if these two lines should be deleted.

Reply:
removed it